src/main/java/io/aiven/kafka/auth/json/reader/AclJsonReader.java

```diff
@@ -39,7 +39,11 @@ public AclJsonReader(final Path configPath) {
     public List<AivenAcl> read() {
         try (final Reader reader = Files.newBufferedReader(configFile)) {
             final java.lang.reflect.Type t = new TypeToken<Collection<AivenAcl>>() {}.getType();
-            return gsonBuilder.create().fromJson(reader, t);
+            final List<AivenAcl> acls = gsonBuilder.create().fromJson(reader, t);
+            if (acls != null) {
+                acls.forEach(AivenAcl::validate);
```

Comment:
Is this `validate` now called twice for each, first in constructor and now here?